binr/ragg2/ragg2.c

```diff
@@ -400,6 +405,14 @@ int main(int argc, char **argv) {
 		}
 	}
 
+	// append the provided string
+	if (str) {
+		int l = strlen(str);
```

Comment:
missing space before (

Reply:
Okay my bad, I'll fix it.
Thank you for your review :)